packages/forms/resources/views/components/date-time-picker.blade.php

```diff
@@ -73,14 +78,25 @@
                 ])
             />
 
-            <span class="absolute inset-y-0 right-0 flex items-center pr-2 pointer-events-none rtl:right-auto rtl:left-0 rtl:pl-2">
-                <svg @class([
-                    'w-5 h-5 text-gray-400',
-                    'dark:text-gray-400' => config('forms.dark_mode'),
-                ]) xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M8 7V3m8 4V3m-9 8h10M5 21h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v12a2 2 0 002 2z" />
-                </svg>
-            </span>
+            @if ($icon)
+                <span class="absolute inset-y-0 right-0 flex items-center pr-2 pointer-events-none rtl:right-auto rtl:left-0 rtl:pl-2">
+                    @if ($icon === 'default')
+                        <svg @class([
+                            'w-5 h-5 text-gray-400',
+                            'dark:text-gray-400' => config('forms.dark_mode'),
+                        ]) xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Comment:
Please put each attribute on their own line.

```suggestion
                        <svg @class([
                            'w-5 h-5 text-gray-400',
                            'dark:text-gray-400' => config('forms.dark_mode'),
                        ]) xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```